x-pack/plugin/eql/src/main/resources/org/elasticsearch/xpack/eql/plugin/eql_whitelist.txt

```diff
@@ -49,6 +49,7 @@ class org.elasticsearch.xpack.ql.expression.function.scalar.whitelist.InternalQl
 # Regex
 #
   Boolean regex(String, String)
+  Boolean regex(String, String, Boolean)
```

Comment:
Do we need that also in the `eql_whitelist`
Since we use `{ql}`: https://github.com/elastic/elasticsearch/pull/68791/files#diff-86dbefecc994e6c82a4e4dc0346417f6a15dab7afc7078ae59196cb4bac854d9R84 ?

Reply:
This is inside Eql. As for SQL that was applied there as well - notice it's on the InternalQlScripts class, the base common one.

Reply:
thx!